app/views/conversations/profile.html.erb

```diff
@@ -0,0 +1,25 @@
+<% presenter = ConversationPresenter.new(@conversation) %>
+<div class="card">
+  <div class="card-body">
+    <div class="card-title">
+      <div class="row">
+        <h1 class="col">
+          <%= presenter.emoji %> 
+        </h1>
+        <div class="col">
+          <% if current_user %>
+            <%= link_to 'Back', @conversation, class: 'btn btn-secondary float-right' %>
+          <% else %>
+            <%= link_to 'Back', root_path, class: 'btn btn-secondary float-right' %>
+          <% end %>
```

Comment:
I'm not sure but maybe a ternary operator will work, I think this because only change the path

Reply:
It's correct, you can change the path with a ternary operator

Reply:
I'll try that, thanks!